x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ilm/TimeseriesLifecycleTypeTests.java

```diff
@@ -166,6 +166,9 @@ public void testValidateColdPhase() {
         } else {
             TimeseriesLifecycleType.INSTANCE.validate(coldPhase.values());
         }
+        if (actions.containsKey(FreezeAction.NAME)) {
+            assertSettingDeprecationsAndWarnings(new String[0], TimeseriesLifecycleType.FREEZE_ACTION_DEPRECATION_WARNING);
```

Comment:
Super minor, but we have `Strings.EMPTY_ARRAY` for this also if needed (it doesn't really matter much here)